CONTRIBUTING.md

```diff
@@ -196,6 +198,44 @@ For both `babel-plugin-x` and `babylon`, you can easily generate an `expected.js
             - expected.json (will be generated if not created)
 ```
 
+#### Debugging code
+
+A common approach to debugging JavaScript code is to walk through the code using the [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/) debugger.
+For illustration purposes, we are going to assume that we need to get a better understanding of [`Generator.generate()`](https://github.com/babel/babel/blob/master/packages/babel-generator/src/index.js#L32), which is responsible for generating code for a given AST.
```

Comment:
We'll probably want to use an exact sha for this (if you go to that link and press `y`): https://github.com/babel/babel/blob/2974a82df16172dc490f868216b121891c4e4bfd/packages/babel-generator/src/index.js#L32